egs/mgb2/asr/local/score_combine.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done